src/interactions/map_city/enterMapForest.interactions.js

```diff
@@ -0,0 +1,9 @@
+import { displayDialogueWithoutCharacter } from '../../utils';
```

Comment:
Please remove the import here because it is not used. Also, the method is now called displayDialogue

Reply:
Done! thank you, must've missed that too